stacker/lookups/__init__.py

```diff
@@ -15,7 +15,7 @@
                                        # space
 ?\s*                                   # any number of spaces separating the
                                        # type from the input
-(?P<input>[@\+\/,\._\-a-zA-Z0-9\:\s=]+) # the input value to the lookup
+(?P<input>[@\+\/,\._\-a-zA-Z0-9\:\s=\[\]\*]+) # the input value to the lookup
```

Comment:
Honestly, @ralph-tice, I'd like to do away with that regex someday.  I've attempted to refactor the way we parse lookups in configs a few times now, and it's surprisingly hard for a few reasons:

- we allow nested lookups
- we allow multiple lookups per variable
- we allow lookups to return non-text values (basically any python object)
- lookups can actually affect the way the external code works (only the `output` lookup at this point, which is used to determine dependencies)

This results in it being shockingly complex to use one of the existing parsers out there.  At some point I'll give it another shot and hopefully be more successful.